basic-cradle-manager/src/main/java/com/exactpro/cradle/module/DummyCradleManager.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020-2022 Exactpro (Exactpro Systems Limited)
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.cradle.module;
+
+import com.exactpro.cradle.CradleManager;
+import com.exactpro.cradle.CradleStorage;
+
+public class DummyCradleManager extends CradleManager {
```

Comment:
If it is base implementation provided configuration then rename to BaseCradleManager otherwise please explain the goal of this class